app/models/declaration/governmentagencygoodsitem/GovernmentAgencyGoodsItem.scala

```diff
@@ -90,23 +87,4 @@ case class GovernmentAgencyGoodsItemAdditionalDocumentSubmitter(
 
 case class DateTimeElement(dateTimeString: DateTimeString)
 
-case class DateTimeString(formatCode: String, value: String) {
-  import DateTimeFormats._
-
-  def time(): ZonedDateTime = formatCode match {
-    case "102" => handle102Format(value)
-    case "304" => handle304Format(value)
-  }
-}
-
-object DateTimeFormats {
```

Comment:
Why this glue-code goes out?

Reply:
I checked for usages and couldn't find any

Reply:
Those formats was related with wco-dec. Hopefully now we're handling date in different (correct) way. Those two formats are formats that you can find in your declaration. I don't remember why we handled this in way above but I'm sure it was related with wco-dec

Reply:
We persisted wco models in the cache.. so I think these formats were used to convert the dates before they were cached.